### PR TITLE
Add Android screenshot info with screengrab

### PR DESCRIPTION
**Pull request by mfurtak**
I will incorporate _supply_ info in a follow-up PR

### CODE REVIEW

File: docs/getting-started/android/screenshots.md [hunk 1]
```diff
@@ -1,7 +1,123 @@
 # fastlane screenshots for Android
 
-To be written soon, this will contain
+Your app screenshots are probably the most important thing when it comes to convincing potential users to download or purchase your app. Unfortunately, many apps don't do screenshots well. This is often because taking screenshots of your app and then preparing them for display is time consuming, and it's hard to get everything right and consistent! For example:
 
-- Getting started with _screengrab_
-- Uploading the screenshots using _supply_
+- Are the screenshots in the App Store inconsistent with your latest app design?
+- Is your app localized into many languages that require different screenshots for each?
+- Have you made sure that no loading indicators are showing?
```
Comment (KrauseFx) [inline docs/getting-started/android/screenshots.md]: Is that a thing on Android too?

**Comment by KrauseFx** [comment]
This looks great to me. Ready for merge?

File: docs/getting-started/android/screenshots.md (hunk 1 above)
Comment (mfurtak) [inline docs/getting-started/android/screenshots.md]: Not exactly, and I removed a bunch of other references to such, so I just missed this one!

Will remove

File: docs/getting-started/android/screenshots.md [hunk 2]
```diff
@@ -1,7 +1,123 @@
 # fastlane screenshots for Android
 
-To be written soon, this will contain
+Your app screenshots are probably the most important thing when it comes to convincing potential users to download or purchase your app. Unfortunately, many apps don't do screenshots well. This is often because taking screenshots of your app and then preparing them for display is time consuming, and it's hard to get everything right and consistent! For example:
 
-- Getting started with _screengrab_
-- Uploading the screenshots using _supply_
+- Are the screenshots in the App Store inconsistent with your latest app design?
```
Comment (asfalcone) [inline docs/getting-started/android/screenshots.md]: in Google Play vs App Store?

File: docs/getting-started/android/screenshots.md (hunk 1 above)
Comment (KrauseFx) [inline docs/getting-started/android/screenshots.md]: 👍

**Comment by mfurtak** [comment]
Ready to go!

File: docs/getting-started/android/screenshots.md [hunk 3]
```diff
@@ -1,7 +1,122 @@
...
+Screengrab.setDefaultScreenshotStrategy(new UiAutomatorScreenshotStrategy());
+```
+
+# Upload Screenshots to Google Play
+
+After generating your screenshots using `fastlane screengrab`, you'll usually want to upload them to Google Play.
+
+**TODO: Bring in _supply_ getting started content**
+
+To upload the screenshots stored in `fastlane/screenshots`, just run
```
Comment (asfalcone) [inline docs/getting-started/android/screenshots.md]: The screenshots are stored in `fastlane/metadata/android` not fastlane/screenshots

Comment (mfurtak) [inline docs/getting-started/android/screenshots.md]: Thanks! 👍

**Pull request merged**